src/app/components/ScrollablePromo/Promo/index.jsx

```diff
@@ -107,6 +119,7 @@ Promo.propTypes = {
     }).isRequired,
   }).isRequired,
   onClick: func.isRequired,
+  isRecommendationType: bool.isRequired,
```

Comment:
Is there a reason why this prop needs to be required? Can it not just be optional, so we don't have to provide this boolean for the exisiting non-recommendation usage

Reply:
We won't need to provide this boolean for the existing non-recommendation usage. We will always have a boolean value passed to the Promo component that comes from the SrollabePromo/index.jsx. In that file we Default the value to false if we don't provide the boolean value.

Reply:
Essentially, Default will never be used, I guess it just comes down to which one we prefer for code clarity. I can take out `isRequired` in favour of Defaulting the value to false if that makes more sense :)